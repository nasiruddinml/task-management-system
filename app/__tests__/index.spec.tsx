import { render, screen } from '@testing-library/react';
import Home from '@app/page';

describe('Home', () => {
  it('renders a text', () => {
    render(<Home />);

    const heading = screen.getByText('Go to dashboard');

    expect(heading).toBeInTheDocument();
  });

  it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
