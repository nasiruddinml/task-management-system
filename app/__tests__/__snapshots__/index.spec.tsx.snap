// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <main
    class="main"
  >
    <button
      class="button link"
    >
      Go to dashboard
    </button>
  </main>
</div>
`;
